sample/src/main/AndroidManifest.xml

```diff
@@ -50,6 +50,17 @@
         <category android:name="android.intent.category.LAUNCHER"/>
       </intent-filter>
     </activity>
+    <activity
+        android:name=".ArchComponentActivity"
+        android:label="@string/app_name_kotlin"
```

Comment:
Let's use a different name for this to avoid confusion. Definitely starting to think we should have a central activity that opens up to these other ones

Reply:
My bad, should've changed the name.
Definitely think we should have a central activity that opens up to the other ones. Spent 5 minutes yesterday wondering why the ArchComponentActivity didn't log correctly. Then noticed I hadn't commented out the MainActivity 😄 

Do you want to include that change in this PR?

Reply:
I'll include this in another PR